docs/usage/other_features/cron_scheduler.md

```diff
@@ -45,7 +45,7 @@ You can set the cron schedule by filling in this form. Please see [Cron Trigger
 
 You can set the cron executing user by filling in this form and press the enter key.
 
-### auto-restart interpreter on cron execution
+### After execution close the interpreter
```

Comment:
maybe "stop" is better than "close"? I think that's the terminology used elsewhere
https://zeppelin.apache.org/docs/0.7.3/manual/interpreters.html#what-is-interpreter-group